randovania/data/json_data/prime3.txt

```diff
@@ -1988,6 +2341,13 @@ Temple Generator
 Asset id: 17919508762406024111
 > Dock to Temple Hall; Heals? False
   * Normal Door to Temple Hall/Dock to Temple Generator
+  > Event - Leviathan Shield
+      Ship Grapple
+
+> Event - Leviathan Shield; Heals? False
+  * Event Fire Leviathan Shield Generator
+  > Dock to Temple Hall
+      Trivial
 
 ----------------
 Cavern Entry
```

Comment:
Not for this room, again I can't comment on what wasn't changed.

For Temple Reservoir (below), either node should have requirements ice missile or damage. SA and suit arent needed.

Reply:
Wouldn't "ice missile or damage or hazard shield" work as requirements then? So Hazard Shield would technically be an alternative to taking damage

Reply:
It could but there's really no need for it. It is equivalent to having requirements in Prime 2 to negate poison water damage with Light Suit. You don't need items for it.